server/api/db/sqldb/db.py

```diff
@@ -2514,6 +2514,16 @@ def create_project(self, session: Session, project: mlrun.common.schemas.Project
         labels = project.metadata.labels or {}
         update_labels(project_record, labels)
 
+        object_to_store = [project_record]
+
+        project_summary = self._generate_project_summary(project)
+        if project_summary:
+            object_to_store.append(project_summary)
```

Comment:
```suggestion
        self._generate_project_summary(project, object_to_store)
```
You can append in the function and then there's no need for the if